test_common/harness/errorHelpers.cpp

```diff
@@ -690,21 +691,19 @@ const char *subtests_to_skip_with_offline_compiler[] = {
     "library_function"
 };
 
-int check_functions_for_offline_compiler(const char *subtestname,
-                                         cl_device_id device)
+bool check_functions_for_offline_compiler(const char *subtestname)
 {
     if (gCompilationMode != kOnline)
     {
         size_t nNotRequiredWithOfflineCompiler =
```

Comment:
The merge conflict what on this line: `int` was also replaced with `size_t` in https://github.com/KhronosGroup/OpenCL-CTS/commit/6c8045911ab193143eae48eef68fc966d0d96b1f#diff-55abc7e642eddea2d060e0885ee8c5105d3a0e9af3e85d4de04cd29112c76845.